CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 
 # Loki operator
 /operator/ @grafana/loki-team @periklis @xperimental
+/.github/workflows/operator* @grafana/loki-team @periklis @xperimental
```

Comment:
`operator*` because, looks like they do support `glob` pattern.
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax